lib/galaxy/dependencies/update.sh

```diff
@@ -29,10 +29,14 @@ done
 shift $((OPTIND - 1))
 
 # Create a virtual environment in a tmp directory and install uv into it
-uv_venv=$(mktemp -d "${TMPDIR:-/tmp}/uv_venv.XXXXXXXXXX")
-python3 -m venv "${uv_venv}"
-"${uv_venv}/bin/python" -m pip install uv
-uv="${uv_venv}/bin/uv"
+if command -v uv >/dev/null; then
+    uv="$(command -v uv)"
```

Comment:
The idea of installing `uv` from scratch was to have it at the last version, so maybe:

```suggestion
    uv="$(command -v uv)"
    ${uv} self update
```